src/views/distribusiRdt/list.vue

```diff
@@ -0,0 +1,278 @@
+                  <td>{{ item.name.toUpperCase() }}</td>
+                  <td>{{ Math.abs(item.quantity) }}</td>
+                  <td>{{ item.time.substr(0, 10) }}</td>
+                  <td><a href="">Edit</a></td>
+                </tr>
+              </tbody>
+            </template>
+          </v-simple-table>
+        </v-col>
+      </v-row>
+    </v-card>
+    <pagination
+      :total="totalList"
+      :page.sync="listQuery.page"
+      :limit.sync="listQuery.limit"
+      :on-next="onNext"
+    />
+    <dialog-delete
+      :dialog="dialog"
+      :data-deleted="dataDelete"
+      :dialog-delete.sync="dialog"
+      :delete-date.sync="dataDelete"
+      :store-path-delete="`reports/deleteReportCase`"
+      :store-path-get-list="`rdtDistribution/getListRdtDistribution`"
+      :list-query="listQuery"
+    />
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+export default {
+  name: 'LaporanList',
+  data() {
+    return {
+      totalReport: 0,
+      queryReportCase: {
+        address_district_code: ''
+      },
+      listQuery: {
+        sort: 'asc',
+        page: 1,
+        limit: 10,
+        search: '',
+        kabkota_kode: '',
+        kec_kode: '',
+        time: ''
+      },
+      countingReports: null,
+      dialog: false,
+      dataDelete: null,
+      modal: false,
+      date: null,
+      sortOptions: [
+        { value: 'asc', label: 'A-Z' },
+        { value: 'desc', label: 'Z-A' }
+      ]
+    }
+  },
+  computed: {
+    ...mapGetters('rdtDistribution', [
+      'listRdtDistribution',
+      'totalList',
+      'firstStock',
+      'distributedStock',
+      'remainingStock'
+    ]),
+    ...mapGetters('user', [
+      'roles',
+      'district_user'
+    ])
+  },
+  watch: {
+    'listQuery.search': {
+      handler: function(value) {
+        if (value.length >= 3) {
+          this.listQuery.page = 1
+          this.handleSearch()
+        } else if (value.length === 0) {
+          this.listQuery.page = 1
+          this.handleSearch()
+        }
+      },
+      immediate: true
+    }
+  },
+  async mounted() {
+    this.listQuery.address_district_code = this.district_user
+    this.queryReportCase.address_district_code = this.district_user
+    await this.$store.dispatch('rdtDistribution/getSummary')
+    this.handleSearch()
+  },
+  methods: {
+    async handleSearch() {
+      await this.$store.dispatch('rdtDistribution/getListRdtDistribution', this.listQuery)
+    },
+    getTableRowNumbering(index) {
+      return ((this.listQuery.page - 1) * this.listQuery.limit) + (index + 1)
+    },
+    async onNext() {
+      await this.$store.dispatch('rdtDistribution/getListRdtDistribution', this.listQuery)
+    },
+    changeDate(value) {
+      this.listQuery.time = value
+      this.handleSearch()
+    },
+    changeDistrict(value) {
+      !value ? this.listQuery.kabkota_kode = '' : this.listQuery.kabkota_kode = parseFloat(value.kemendagri_kabupaten_kode)
```

Comment:
Bukan seperti itu, tapi kayak gini. Cobain test, jalan ga

```suggestion
      this.listQuery.kabkota_kode = !value ? '' : parseFloat(value.kemendagri_kabupaten_kode)
```

Reply:
udah dibenerin bisa pak